observability/network_observability/installing-operators.adoc

```diff
@@ -5,10 +5,14 @@ include::_attributes/common-attributes.adoc[]
 :context: network_observability
 
 toc::[]
-Installing Loki is a recommended prerequisite for using the Network Observability Operator. You can choose to use xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki], but there are some considerations for doing this, described in the previously linked section.
 
 The {loki-op} integrates a gateway that implements multi-tenancy and authentication with Loki for data flow storage. The `LokiStack` resource manages Loki, which is a scalable, highly-available, multi-tenant log aggregation system, and a web proxy with {product-title} authentication. The `LokiStack` proxy uses {product-title} authentication to enforce multi-tenancy and facilitate the saving and indexing of data in Loki log stores.
 
+[NOTE]
+====
+Installing Loki is a recommended prerequisite for using the Network Observability Operator. You can choose to use xref:../../observability/network_observability/installing-operators.adoc#network-observability-without-loki_network_observability[Network Observability without Loki], but there are some considerations for doing this, described in the previously linked section.
+====
```

Comment:
Let's take this out of the admonition and put it back like it was. I don't love the fact that now the first paragraph of this installation Assembly is completely about Loki and the Using the Network Observability operator without Loki is in the NOTE. Sometimes people gloss over notes as "unimportant" and the "Network Observability without Loki" piece is pretty important for us in a "promoting usership" of the Operator sense. 

cc @gwynnemonahan